### PR TITLE
Issue 1608 mask composite error on nonsquare masks

### PR DESCRIPTION
**Pull request by rsamf**
See #1608 

I was having problems with using the MaskComposite node for non-square masks. I believe this is the fix. Probably somewhere down the line of development the dimensions of the masks became flipped.

### CODE REVIEW

File: comfy_extras/nodes_mask.py [hunk 1]
```diff
@@ -237,24 +237,24 @@ def combine(self, destination, source, x, y, operation):
         source = source.reshape((-1, source.shape[-2], source.shape[-1]))
 
         left, top = (x, y,)
-        right, bottom = (min(left + source.shape[-1], destination.shape[-1]), min(top + source.shape[-2], destination.shape[-2]))
+        right, bottom = (min(left + source.shape[-2], destination.shape[-2]), min(top + source.shape[-1], destination.shape[-1]))
         visible_width, visible_height = (right - left, bottom - top,)
 
         source_portion = source[:visible_height, :visible_width]
         destination_portion = destination[top:bottom, left:right]
```
Comment (0x473354) [inline comfy_extras/nodes_mask.py]: Lines 243 and 244 are slicing along the wrong dimensions, should be:
```python
        source_portion = source[:, :visible_height, :visible_width]
        destination_portion = destination[:, top:bottom, left:right]
````
That's all I changed locally to fix the issue.

Comment (rsamf) [reply comfy_extras/nodes_mask.py]: Good find! I'll make the change. I wonder why both output shapes work (H,W) and (W,H)

**Comment by mcmonkey4eva** [comment]
Is this still needed?

**Comment by 0x473354** [comment]
The change was already made in https://github.com/comfyanonymous/ComfyUI/commit/b5fa3d28d7a9242a73c40b3b37e526267cbd67fd

**Pull request closed without merging**